dnf/util.py

```diff
@@ -123,6 +123,7 @@ def rtrim(s, r):
 
 
 def am_i_root():
+    # :api
```

Comment:
Not sure if we really want to make it as API anything here - dnf/util.py

Reply:
Not marked as API any more. I just added comment that ansible is using these functions.